src/main/java/org/fundacionjala/coding/dennis/FizzBuzz.java

```diff
@@ -0,0 +1,53 @@
+package org.fundacionjala.coding.dennis;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ */
+public  final class FizzBuzz {
+
+    public static final int THREE = 3;
+    public static final int FIVE = 5;
+    public static final int MCM = THREE * FIVE;
+    public static final int LIMIT = 200;
+    /**
+     *
+     */
+    private FizzBuzz() {
+    }
+
+    /**
+     *
+     * @param limit length of the list
+     * @return List StringBuilders.
+     */
+
+    //Part 1
+    public static List<StringBuilder> fBuzz(int limit) {
+        List<StringBuilder> fbuzz = new ArrayList<>();
+        for (int pt = 1; pt <= limit; pt++) {
+            if (pt % MCM == 0) {
+                fbuzz.add(new StringBuilder("FizzBuzz"));
+            } else if (pt % THREE == 0) {
+                fbuzz.add(new StringBuilder("Fizz"));
+            } else if (pt % FIVE == 0) {
+                fbuzz.add(new StringBuilder("Buzz"));
+            } else {
+                fbuzz.add(new StringBuilder(Integer.toString(pt)));
+            }
+        }
+
+        //Part 2
+        List<StringBuilder> fbuzz2 = new ArrayList<>();
+        for (StringBuilder elem : fbuzz) {
+            StringBuilder aux = new StringBuilder(elem.toString().replaceAll("[^3]", ""));
+            elem = aux.length() < 1 ? elem : new StringBuilder("Fizz");
+            aux = new StringBuilder(elem.toString().replaceAll("[^5]", ""));
+            elem = aux.length() < 1 ? elem : new StringBuilder("Buzz");
+            fbuzz2.add(elem);
+        }
+        return fbuzz2;
```

Comment:
@AT-08/at-08 
I recommend you to have two methods 
* one of them given a single number -> return/convert to fizz buzz covering the parts of the kata.
* the another one to print the numbers from 1 to 100 as fizz buzz.